src/Honeybee_ExportEPC.py

```diff
@@ -1,20 +1,29 @@
-# By "Write your names here..."
-# "Write your email address here
+# By Patrick Chopson and Sandeep Ahuja
+# patrick.chopson@perkinswill.com and sandeep.ahuja@perkinswill.com
 # Honeybee started by Mostapha Sadeghipour Roudsari is licensed
 # under a Creative Commons Attribution-ShareAlike 3.0 Unported License.
 
+##!!!!!!Download the latest EPC (Monthly) and EPC Weather Converter from - http://zeroenergygreen.com/downloads/
+
 """
 Export to Energy Performace Calculator (EPC)
-... Add more description here
++The EPC is an Excel based energy modeling software.
++Instant results no waiting for calculation results.
++Reduces a building down to “box”
++Developed for 20 years at the Georgia Institute of Technology - Building Technology Lab.
++EPC is proven to be accurate to within 5-6% of more sophisticated tools such as EnergyPlus or eQuest.
++Normative energy calculator defined by ISO 13970 and CEN 15603.
++Monthly and Hourly versions proven superior to dynamic simulation methods in ASHRAE 90.1, Appendix G.
 
 -
 Provided by Honeybee 0.0.55
 
     Args:
         _HBZones: Honeybee Zones
+        filename: Name of the Excel EPC file
+        filepath: File path to Excel EPC file
     Returns:
-        opaqueSrfsArea: Area of the surfaces for each direction
-        glazedSrfsArea: Area of the glazing surfaces for each direction
+        EUI:  Energy Use Intensity in kWhr/m^2/yr 
```

Comment:
Make sure to update ghenv.Component.Message for the current date and current version. Line 31 in the code.